frontend/src/views/HomeView.vue

```diff
@@ -2,16 +2,66 @@
   <div class="mt-5">
     <SloganComponent />
     <div class="mt-5">
-      <DropToStartField />
+      <DropToStartField @dropFilesHandler="dropFilesHandler" />
     </div>
     <div class="mt-5">
       <FrequentlyAskedQuestions />
     </div>
   </div>
+  <div v-if="hasErorr" class="alert alert-danger alert-dismissible" role="alert">
+    <div>{{ errorMessage }}</div>
+    <button type="button" @click="dismissAlert" class="btn-close" aria-label="Close"></button>
+  </div>
 </template>
 
 <script setup lang="ts">
 import SloganComponent from '@/components/SloganComponent.vue'
 import DropToStartField from '@/components/DropToStartField.vue'
 import FrequentlyAskedQuestions from '@/components/FrequentlyAskedQuestions.vue'
+import { ref } from 'vue'
+import axios from 'axios'
+import router from '@/router'
+
+const hasErorr = ref(false)
+const errorMessage = ref('')
+
+function dismissAlert() {
+  hasErorr.value = false
+}
+
+async function dropFilesHandler(event: DragEvent) {
+  if (event.dataTransfer?.files == undefined) {
+    return
+  }
+  const files = event.dataTransfer.files
+  axios
+    .post(`${import.meta.env.VITE_API_URL}/users`)
+    .then(() => {
+      const data = {
+        name: 'First Project'
+      }
+      return axios.post(`${import.meta.env.VITE_API_URL}/projects`, data)
+    })
+    .then((response) => {
+      const project_id = response.data.id
+      const uploadFileRequestsList = []
+      for (let i = 0; i < files.length; ++i) {
+        const file = files[i]
+        const requestData = new FormData()
+        requestData.append('project_id', project_id)
+        requestData.append('uploaded_file', file)
+        uploadFileRequestsList.push(
+          axios.post(`${import.meta.env.VITE_API_URL}/projects/${project_id}/files`, requestData)
+        )
+      }
+      return axios.all(uploadFileRequestsList)
+    })
+    .then(() => {
+      router.push({ path: '/project' })
```

Comment:
This is not what we want to do. What we want is the UI to change (hide home, display the project component) as soon as you've created the list of files for upload.
Due to the requirement of being able to drag files on homepage, we have tobe able to display the project page both on home (upon drag) and on `/projects/{id}`

Reply:
> Due to the requirement of being able to drag files on homepage, we have to be able to display the project page both on home (upon drag) and on /projects/{id}

I don't quite understand here. Does it mean that we need to support things like: delete files, change file metadata, request a zim file etc on the Home page?

Reply:
Yes. But since it's a Vue App, it doesn't matter much. All of this will be in a dedicated ProjectsView component (itself composed of several components).
Upon drag, after building the list of files, the HomeView is hidden and the project view is shown

Reply:
My vision is that box on Front Page that prompts to drag and drop files is just an entry point. After the user drags and drops the file, we should go to the Project screen, then automatically create a new project, and then display the results of the user's file upload. 
That is to say, we should enter this interface
<img width="655" alt="image" src="https://github.com/openzim/nautilus-webui/assets/16664654/032f4aff-b30f-481c-ac5c-7582e1388606">

Reply:
Exactly. Except we don't “go to” that page but build it otherwise we either:
- loose the dragged files
- have to wait an eternity in the home page until we can move

Reply:
I shared your wish of going to the project page but @Popolechien and @kelson42 insisted on being able to drag and start rigght from the homepage. That's probably in the design discussion

Reply:
OK I get it. I will put the `DropToStartField` to the dedicated `ProjectsView` and enrich it later. This PR should not include this implementation of this view.

Reply:
👍